### PR TITLE
[SW-450] not found page

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/icons/bug.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pstachel) [inline src/icons/bug.svg:1]:
```suggestion
<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: src/icons/bug.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...3 8.54956 3.66949 7.96657 4.07416C7.93227 4.04672 7.89798 4.01929 7.86369 3.98499C7.57562 3.74494 7.5619 3.42944 7.78138 3.07278L8.12432 2.5378C8.20662 2.39377 8.22034 2.31833 8.22034 2.18801C8.22034 1.89309 7.994 1.66675 7.68536 1.66675C7.45216 1.66675 7.2807 1.77649 7.14352 2.03712L6.74572 2.70241C6.29304 3.5186 6.4165 4.25248 7.12295 4.76003L7.19839 4.8149C6.69771 5.41846 6.38221 6.20721 6.38221 7.05769C6.38221 8.11394 7.04064 8.8341 8.2272 9.05358V9.12217C8.04201 9.24562 7.9117 9.43081 7.82939 9.63657L6.32734 9.44453C5.85408 9.38966 5.57288 9.21819 5.57288 8.69693V6.5913C5.57288 6.22093 5.36026 6.00145 4.98989 6.00145C4.61952 6.00145 4.40004 6.22093 4.40004 6.5913V8.94384C4.40004 9.85605 4.81842 10.391 5.74435 10.5076L7.89112 10.782C7.98028 11.0632 8.17233 11.3238 8.44668 11.4541V11.5227L5.33282 12.2429C4.38632 12.4555 4.03653 12.9288 4.1737 13.7998L4.72926 17.332C4.79098 17.7093 4.96245 17.9425 5.33968 17.9425C5.69633 17.9425 5.97068 17.6887 5.9021 17.2497L5.40827 14.0193C5.34654 13.6078 5.48371 13.4157 5.94325 13.2785C6.11472 13.2305 6.28618 13.1894 6.45765 13.1414C6.35477 13.3609 6.30676 13.6146 6.30676 13.9233C6.30676 16.2072 8.02144 18.3334 9.99675 18.3334Z" fill="white"/>
```
Comment (pstachel) [inline src/icons/bug.svg:2]: change`fill="white"` to `fill="currentColor"`

File: src/popup/router/routes.js [hunk 3]
```diff
@@ -508,7 +508,11 @@ export default [
     path: '*',
     component: NotFound,
     meta: {
-      ifNotAuth: true,
+      asModal: true,
+      backButton: true,
+      title: 'not-found',
+      closeButton: true,
+      hideNotificationsIcon: true,
```
Comment (pstachel) [inline src/popup/router/routes.js:515]: I suggest grouping the keys 
```suggestion
      title: 'not-found',
      asModal: true,
      backButton: true,
      closeButton: true,
      hideNotificationsIcon: true,
```

**Review by pstachel — CHANGES_REQUESTED** [review]

**Review by pstachel — APPROVED** [review]

File: src/popup/pages/NotFound.vue [hunk 4]
```diff
@@ -1,23 +1,95 @@
...
+
+  .title {
+    margin-top: 16px;
+
+    @extend %face-sans-22-bold;
+  }
+
+  .description {
+    margin-top: 8px;
+    margin-bottom: 24px;
```
Comment (CedrikNikita) [inline src/popup/pages/NotFound.vue:78]:
```suggestion
    margin-bottom: 32px;
```

File: src/popup/locales/en.json [hunk 5]
```diff
@@ -416,8 +417,10 @@
       "urlToClaim": "Claim tips from this URL:"
     },
     "notFound": {
-      "notFound": "Requested page can't be found",
-      "initialPage": "Open initial page"
+      "notFound": "Lost in space",
```
Comment (CedrikNikita) [inline src/popup/locales/en.json:420]: Question mark is missing.

File: src/popup/pages/NotFound.vue [hunk 6]
```diff
@@ -1,23 +1,95 @@
...
+      :src="Image"
+    >
+    <div class="title">
+      {{ $t('pages.notFound.notFound') }}
+    </div>
+    <div class="description">
+      {{ $t('pages.notFound.description') }}
+    </div>
+    <BtnMain
+      class="button"
```
Comment (CedrikNikita) [inline src/popup/pages/NotFound.vue:14]: Actually you can use a `btn-main` class to refer that element and not set additional class. But that is something to be defined
https://github.com/aeternity/superhero-wallet/issues/1676

Comment (peronczyk) [reply src/popup/pages/NotFound.vue:14]: I'm opting for leaving the additional classes :)
It's more clear for the component internal code and makes the further refactors easier.
But yes, the conversation on standardising this is in progress. For now I would suggest to leave the decision to the code author.

File: src/popup/router/routes.js [hunk 7]
```diff
@@ -508,7 +508,11 @@ export default [
     path: '*',
     component: NotFound,
     meta: {
-      ifNotAuth: true,
```
[the remark this answers is not in the document]
Comment (peronczyk) [reply src/popup/router/routes.js:511]: I think the change makes sense. If not authorized user will try to open not existing feature he needs to be moved to the index.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/pages/NotFound.vue (hunk 6 above)
Comment (konradodo) [reply src/popup/pages/NotFound.vue]: I suggest leaving this solution

**Comment by CedrikNikita** [comment]
Commit message is saying "feat: add not found page", but that's is not true, since we already had a not found page.

**Comment by CedrikNikita** [comment]
There are a close button in the designs.

**Review by peronczyk — APPROVED** [review]

**Pull request merged**